tests/js/web/NavMainComponentTest.js

```diff
@@ -55,173 +58,271 @@ describe('NavMain component', function() {
+            expect(this.$html.find('.nav-primary .nav-items [label="More"]').is(':visible')).to.be.true;
+        });
+
+        describe('when the more link is clicked', function () {
+            beforeEach(function() {
+                this.$moreIconLink = this.$navMain.find('.more-icon > .nav-link');
+                this.$moreIconLink.click();
+            });
+
+            it('should open the sliding main nav if it is closed', function () {
+                expect(this.$navTertiary.hasClass('is-open')).to.be.true;
+            });
+        });
+    });
+
+    describe('clicking the close icon, when the tertiary sub navigation is open', function () {
+        beforeEach(function () {
+            this.navMainComponent.init();
+            this.$secondaryLinkThree.click();
+            this.$closeLink.click();
+        });
 
+        it('should close the tertiary sub navigation', function () {
+            expect(this.$html.find('.nav-tertiary').hasClass('is-open')).to.be.false;
+        });
     });
 
+    describe('clicking the close icon, when the quaternary sub navigation is open', function () {
+        beforeEach(function() {
+            this.navMainComponent.init();
+            this.$secondaryLinkFour.click();
+            this.$closeLink.click();
+        });
+
+        it('should close the quaternary sub navigation', function () {
+            expect(this.$html.find('.nav-quaternary').hasClass('is-open')).to.be.false;
+        });
+    });
+
+    describe("When the window is resized from very short to large enough to fit nav", function () {
+        beforeEach(function () {
+            this.navMainComponent.init();
+            this.$window.height(200);
+            this.$window.resize();
+        });
+
+        it('should hide the More link', function () {
+            expect(this.$html.find('.nav-primary .nav-items [label="More"]').is(':visible')).to.be.false;
+        });
+
+        it('should show extra nav items', function () {
+            expect(this.$html.find('.nav-primary .nav-items li:not([label="More"])').is(':visible')).to.be.true;
+        });
+    });
+
+    describe("When nav-items is bigger than the window and More Item exists", function () {
+        beforeEach(function () {
+            this.navMainComponent.init();
+            this.$window.resize();
+        });
+
+        it('should show the More link', function () {
+            expect(this.$html.find('.nav-primary .nav-items [label="More"]').is(':visible')).to.be.true;
+        });
+    });
+
+    describe('When the window is resized', function () {
+        beforeEach(function () {
+            this.adjustNavStub = sinon.stub(this.navMainComponent, 'adjustNavItems');
+        });
+
+        it('should call adjustNavItems()', function () {
+            this.navMainComponent.init();
+            this.navMainComponent.$window.trigger('resize');
+            expect(this.adjustNavStub).to.have.been.called;
```

Comment:
Channeling  my inner @asmblah here, could we have a line break before the `expect(...)` line